test/e2e/hello-world-encrypted-HW.json

```diff
@@ -1,19 +1,19 @@
 {
   "metadata": {
-    "name": "hello-world-encrypted-container"
+    "name": "ccv0-ssh-encrypted-container"
   },
   "image":{
-    "image": "docker.io/eqmcc/helloworld_enc"
+    "image": "ghcr.io/confidential-containers/test-container:encrypted"
   },
   "envs": [
     {"key": "OCCLUM_RELEASE_ENCLAVE", "value": "1"}
   ],
   "command": [
     "/run/rune/boot_instance/build/bin/occlum-run",
-    "/bin/hello_world"
+    "/usr/sbin/sshd"
```

Comment:
is the image running OK? The logs report:
```
sshd re-exec requires execution with an absolute path
["init"]
```

Reply:
No. It's not ok. `sshd` requires absolute path to run like `/usr/sbin/sshd` rather than `./sshd`. This might be caused by `occlum-run`?  Could you help to give any comments? @qzheng527

Reply:
This is because occlum will change argv[0] from a absolute path into the binary name here https://github.com/occlum/occlum/blob/master/src/run/main.c#L24 . The error is not related to enclave-cc, but it seems that we can not realize a successfully running `ccv0-sshd` service via enclave-cc for v0.5.0.

Two choices for us:
1. Use the previous `hello_world` instead
2. Use this image, and treat it is launched "successfully"

Reply:
> I'd prefer option 1.

I have pushed the encrypted image (hello-world) to `ghcr.io`. However, I do not have permission to make it public. Let me ask Tobin for help and we can retrigger HW test then

Reply:
@Xynnn007 OK, thanks! I think the PR looks good otherwise.